### PR TITLE
Waitlist limit

### PR DESCRIPTION
**Pull request by HooriyaHK**

### CODE REVIEW

File: code/app/src/main/res/values/refs.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zyzCodes) [inline code/app/src/main/res/values/refs.xml:1]: this file is extra please delete it

**Review by zyzCodes — CHANGES_REQUESTED** [review]
please complete testing, also WaitlistUsersAdapter.java was deleted. Do we want to delete this file?

**Comment by HooriyaHK** [comment]
> please complete testing, also WaitlistUsersAdapter.java was deleted. Do we want to delete this file?

Yes, the recycler adapter does what this does

File: code/app/src/main/res/values/refs.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+<resources></resources>
```
Comment (zyzCodes) [inline code/app/src/main/res/values/refs.xml:2]: do we need this changes to this file?

Comment (zyzCodes) [reply code/app/src/main/res/values/refs.xml:2]: if no, please remove this

**Comment by zyzCodes** [comment]
> > please complete testing, also WaitlistUsersAdapter.java was deleted. Do we want to delete this file?
> 
> Yes, the recycler adapter does what this does

okay, thanks for the clarification

**Review by zyzCodes — APPROVED** [review]
we are good to merge this if we have verified that it updates the limit in firebase

**Pull request merged**